inst/include/Rcpp/exceptions_impl.h

```diff
@@ -0,0 +1,88 @@
+// exceptions_impl.h: Rcpp R/C++ interface class library -- exceptions
+//
+// Copyright (C) 2020 Joshua N. Pritikin
```

Comment:
We should preserve the old author names as well (adding yourself is okay since you've also made changes)